webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/MergedVolume.scala

```diff
@@ -0,0 +1,135 @@
+package com.scalableminds.webknossos.tracingstore.tracings.volume
+
+import java.io.File
+
+import com.scalableminds.util.geometry.Point3D
+import com.scalableminds.util.tools.{ByteUtils, Fox}
+import com.scalableminds.webknossos.datastore.models.{BucketPosition, UnsignedInteger, UnsignedIntegerArray}
+import com.scalableminds.webknossos.datastore.services.DataConverter
+import com.scalableminds.webknossos.tracingstore.VolumeTracing.VolumeTracing.ElementClass
+import com.scalableminds.webknossos.tracingstore.tracings.ProtoGeometryImplicits
+import net.liftweb.common.Box
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+
+class MergedVolume(elementClass: ElementClass, initialLargestSegmentId: Long = 0)
+    extends DataConverter
+    with ByteUtils
+    with VolumeDataZipHelper
+    with ProtoGeometryImplicits {
+  private var mergedVolume = mutable.HashMap.empty[BucketPosition, Array[UnsignedInteger]]
+  private var labelSets = mutable.ListBuffer[mutable.Set[UnsignedInteger]]()
+  private var labelMaps = mutable.ListBuffer[mutable.HashMap[UnsignedInteger, UnsignedInteger]]()
```

Comment:
If we're using mutable types here, we can declare them as `val`s to prevent unwanted new assignments